notebooks/MAST/tess_cutout_rotation_curve/tess_cutout_rotation_curve.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>Can you display superscripts in a notebook? If so, to me, it's easier to ready d^{-1} rather than 1/d.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>